core/src/main/resources/jenkins/model/CauseOfInterruption/UserInterruption/summary_zh_TW.properties

```diff
@@ -22,3 +22,4 @@
 # THE SOFTWARE.
 
 blurb=\u7531\u4f7f\u7528\u8005 <a href="{1}">{0}</a> \u4e2d\u6b62
+userNotFound=\u7531\u4f7f\u7528\u8005 {0} \u4e2d\u6b62
```

Comment:
Where are you getting this from? Best not to add language keys unless you sure fluent.